CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
 # Changelog
 
-## Next Release
+## 20.11.0
 
 ### Features Added
 
 - Interlock/Armory II HSM keystore support
-- Eth2 slashing protection data able to be imported from json file (Interchange format V5)
+- Eth2 slashing protection data able to be exported and imported from json file (Interchange format V5)
+- Eth2 signing API can now return response in json format
+- Genesis validators root validation for eth2 signing requests. The GVR is set on first eth2 signing request.
+- Use a low watermark in Eth2 slashing protection and import/export
+
```

Comment:
* Do not sign below watermark (regardless of if matching existing entry)
* Resolved help text anomalies

Reply:
done